x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/assignees.test.tsx

```diff
@@ -132,12 +132,12 @@ describe('<Assignees />', () => {
     const { getByTestId, getByText } = renderAssignees('test-event', assignees);
 
     // Update assignees
-    getByTestId(ASSIGNEES_ADD_BUTTON_TEST_ID).click();
-    getByText('User 1').click();
-    getByText('User 3').click();
+    fireEvent.click(getByTestId(ASSIGNEES_ADD_BUTTON_TEST_ID));
```

Comment:
[[job]](https://buildkite.com/elastic/kibana-pull-request/builds/266993#0194655f-1467-4bc9-b5e6-fc58957938b9) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/266993/jobs/0194655f-1467-4bc9-b5e6-fc58957938b9/artifacts/0194657e-daba-42b9-9702-f0c36422dfa3) Jest Tests #9 / should call assignees update functionality with the right arguments

Looks like with an update to React@18 (even in legacy mode) these events can be skipped. By using fireEvent we wrap each event in act allowing tests to process pending events.